test/test_fat_fingers.rb

```diff
@@ -147,7 +147,8 @@ def setup
         bad: [
           "test@icoud.com",
           "test@icloud.co",
-          "test@icloud.net"
+          "test@icloud.net",
+          "test@iclod.com"
```

Comment:
I think you can add test cases for `test@iclud.com` and `test@icld.com`, since they are already covered by your changes.

Reply:
Good call. I'll do that.